maps/cynosure/cynosure_defines.dm

```diff
@@ -13,7 +13,7 @@
 	full_name = "Cynosure Station"
 	path = "cynosure"
 
-	lobby_screens = list('icons/default_lobby.png', 'icons/lobby_old.gif') // New lobby screen if possible.
+	lobby_screens = list('maps/cynosure/title_cynosure.png') //Title screen created for Polaris by Adrien Perrier - www.artstation.com/gasgano

```

Comment:
It's probably preferable to use https://github.com/PolarisSS13/Polaris/blob/master/ATTRIBUTIONS.md for this task, instead of a comment

Reply:
Yeah good catch, I'd actually looked for that file in just about every subfolder but am a dumbass and didn't look at the literal root folder very hard.